queues.py

```diff
@@ -0,0 +1,53 @@
+class MyQueue(object):
+
+    def __init__(self):
+        self.queue = []
```

Comment:
we can do this using 2 queues as well
although this is also write way of doing this!!